src/pymedphys/dcm/_level1/dcmdose.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2016-2018 Simon Biggs, Matthew Jennings
+# Copyright (C) 2016-2019 Matthew Jennings and Simon Biggs
```

Comment:
Worth updating the copyright like this when you make a significant contribution to a file.